pkg/controller/apimanager/apimanager_controller.go

```diff
@@ -28,7 +26,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller_apimanager")
+var log = logf.Log.WithName("apimanager.controller")
```

Comment:
The naming scheme seems to be different than what operator-sdk generates for controller log names, which is of the form "controller_<controllername>". We might find it's different on what we might find in most custom controllers around there (for example https://github.com/keycloak/keycloak-operator/blob/bae3b1e49a1a8c8d4008da9735f1db7cb7ea40f5/pkg/controller/keycloak/keycloak_controller.go#L38)

Not

Reply:
```{"level":"info","ts":1588004166.3609462,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"apimanager-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.361117,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"apimanager-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.3612475,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"apimanager-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.3614237,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"binding-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.3615575,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"binding-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.3616793,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"binding-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.3618264,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"binding-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.361956,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"binding-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.362078,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"binding-controller","source":"kind source: /, Kind="}
{"level":"info","ts":1588004166.3622398,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"tenant-controller","source":"kind source: /, Kind="}
```
Which one do you want to use? Up to you.

Reply:
The one in the logs you've shown me is not a controller generated by operator-sdk, which is what I was referring to.

I personally prefer the one with underscore as it's how operator-sdk generates it for you, but both are fine to me so I'll be ok with any decision you make

Reply:
:+1: let it be `controller_apimanager`